test/fixture/certs/baseSVID.pem

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
+MIIDGDCCAgCgAwIBAgIJANH5JfRK7QwQMA0GCSqGSIb3DQEBCwUAMA8xDTALBgNV
+BAMMBHRlc3QwHhcNMTcxMDA1MjAyMTU1WhcNMjcxMDAzMjAyMTU1WjAPMQ0wCwYD
+VQQDDAR0ZXN0MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAx6TldXPO
+ZoT9ZfjcPDfaRmBFxMI8zOXnFrDMh6QHeLNcC815pCK/NKC/ff4KhAaA/wOg/CHN
+D9c9FKz6zY5cFchawrZ76iwWhLsKPD2S7G9UShNopPOIVtz44cMnATisV1jgiEQW
+136ajU1liTwxlHvG6t6kzhFZQ/qykmeuH72VLdsiY7Zl5thLUN3zGlF08D+MPQNA
+ZZhAvoHLGK9ItCU/fgX3cnRQe2XMiHWMsIC/3W806+3hdcjEP2aKB4J9IjgoNdhg
+ioVNAizDxSsJ4wdtetnm8I/c4ZoagdFGNxqwIR0u2sFaSEk7DfxezbeYIphQcgVg
+bdcNqruPjoFJBwIDAQABo3cwdTAJBgNVHRMEAjAAMAsGA1UdDwQEAwIFoDAdBgNV
+HSUEFjAUBggrBgEFBQcDAQYIKwYBBQUHAwIwPAYDVR0RBDUwM4Yxc3BpZmZlOi8v
+ZXhhbXBsZS5vcmcvc3BpcmUvYWdlbnQvam9pbl90b2tlbi90b2tlbjANBgkqhkiG
+9w0BAQsFAAOCAQEAAtvxQ6XaFywQ1GE9hzrOULSX7Zj6sONdutSUmrh07jaWWdts
+clAsQxzG7hNyIhcWk3GT1B7POrHIT6BRBiyZsjtYWpQgHHanfDxa+vXbYIaEOwv1
+nPbhcGsAgHZP73RLaQTQpoG7E3lg2P6WELyel1l7Qti9IdkTZ7rN8RYRvmpb5xrM
+jzQOkQ9Fgsxyb3MSBkdYD15zZxv8ZQ6LQRWKcVRqre3ym8TIiPHpTKwh5bbcraz6
+xTNvTPLU+n+ixjXf+Uu7SYaN9WpH8UPFH0pmi3z+z3mysgh+glW43AUUUTUqaS1m
+U3Y3FCpyg28KaxZQbDp/ucvm2KzFA9i03HtwmA==
+-----END CERTIFICATE-----
```

Comment:
an `svid.pem` has been checked in that you can use or replace

Reply:
yea the rebase pulled in two other pem. will consolidate.